### PR TITLE
Only require vip-jetpack.php when Jetpack file was truly required.

### PR DESCRIPTION
**Pull request by rinatkhaziev**
## Description

Currently, we're requiring `vip-jetpack.php` unconditionally in `vip_jetpack_load`, this PR changes that to only require it the file was successfully required and `VIP_JETPACK_LOADED_VERSION` is set.

The chances of this breaking are slim but not zero, so let's have an additional safe-guard.

### Plugin Updated: Jetpack loader

Do not load Jetpack customizations in case a main Jetpack file was not included properly.

## Checklist

Please make sure the items below have been covered before requesting a review:

- [ ] This change works and has been tested locally (or has an appropriate fallback).
- [ ] This change works and has been tested on a Go sandbox.
- [ ] This change has relevant unit tests (if applicable).
- [ ] This change has relevant documentation additions / updates (if applicable).
- [ ] I've created a changelog description that aligns with the provided examples.

### CODE REVIEW

**Review by WPprodigy — DISMISSED** [review]
Seems sound, I checked the files that end up being loaded unconditionally atm and it doesn't look like there will be any surprises. Any constants/functions/classes are isolated to this portion.

Also, unlikely this happens like you said. If `VIP_JETPACK_LOADED_VERSION` is set then nothing down here would happen anyway.

**Review by pauarge — DISMISSED** [review]

File: jetpack.php [hunk 1]
```diff
@@ -139,7 +139,9 @@ function vip_jetpack_load() {
 		add_filter( 'instagram_cache_oembed_api_response_body', '__return_true' );
 	}
 
-	require_once __DIR__ . '/vip-jetpack/vip-jetpack.php';
+	if ( defined( 'VIP_JETPACK_LOADED_VERSION' ) ) {
```
Comment (rebeccahum) [inline jetpack.php:142]: Should we also check that `VIP_JETPACK_LOADED_VERSION` is not a `none` value? https://github.com/Automattic/vip-go-mu-plugins/blob/b12bf1908960fdf9dcd0e52ccb39edb2d2677e99/jetpack.php#L97-L100

Comment (rinatkhaziev) [reply jetpack.php]: @rebeccahum good catch, added that.

**Review by rebeccahum — APPROVED** [review]
Thanks! 🚢

**Comment by sjinks** [comment]
r2012-stacks

**Comment by sjinks** [comment]
v20211116.0

**Pull request merged**